test-data/unit/fine-grained-modules.test

```diff
@@ -1064,14 +1093,20 @@ def foo() -> None: pass
 [file b.py.2]
 import a
 a.foo(10)
+
+[stale a, b]
```

Comment:
Here and various other places below we don't check for `[rechecked`. Is this intentional?